ui/client/common/ProcessUtils.js

```diff
@@ -9,6 +9,11 @@ class ProcessUtils {
     return !_.isEmpty(fetchedProcessDetails) ? _.isEqual(fetchedProcessDetails.json, processToDisplay) : true
   }
 
+  nothingToExport = (state) => {
```

Comment:
Can we named this method better? Something like availableToExport or just canExport?